api_tests/nodes/views/test_node_wiki_list.py

```diff
@@ -338,7 +338,7 @@ def test_create_public_wiki_page(self, app, user_write_contributor, url_node_pub
         assert res.json['data']['attributes']['name'] == page_name
 
     def test_create_public_wiki_page_with_content(self, app, user_write_contributor, url_node_public, project_public):
-        page_name = fake.word()
```

Comment:
This test was cached as failing because of a 1 in a million coincidence where `fake.word` caused a collision. So this test was passing locally 99% of time, but was frozen in the cache as failing. However I just made the name a static value and obviously that woke up Testmon and it fixed the bug and recached it as passing.

Reply:
interesting!